app/assets/javascripts/copo/copo.edit-checkin.es6

```diff
@@ -87,24 +144,29 @@ window.COPO.editCheckin = {
 
   updateCheckin(response) {
     // tries to find the checkin in gon and update it with the response
-    checkin = _.find(gon.checkins, _.matchesProperty('id',response.id));
-    checkin.lat = response.lat;
-    checkin.lng = response.lng;
-    checkin.edited = response.edited;
+    checkin = _.find(gon.checkins, _.matchesProperty('id', response.checkin.id));
+    checkin.lat = response.checkin.lat;
+    checkin.lng = response.checkin.lng;
+    checkin.edited = response.checkin.edited;
     checkin.lastEdited = true;
-    checkin.address = response.address;
-    checkin.fogged_city = response.fogged_city;
```

Comment:
@t-harps yes, but we should start from [here](https://github.com/earlymarket/Coposition/blob/edit-checkin-datetime-5285/app/models/device.rb#L45)